src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -126,7 +126,7 @@ julia> dim(PF)
 2
 ```
 """
-dim(PF::PolyhedralFan) = pm_object(PF).FAN_DIM
+dim(PF::_FanLikeType) = pm_object(PF).FAN_DIM
```

Comment:
This is not a requirement for this PR, just wanted to point out: Julia can't infer the return type here, so consider adding a type assertion. Based on examples, presumably:
```suggestion
dim(PF::_FanLikeType) = pm_object(PF).FAN_DIM::Int
```